torchdata/nodes/samplers/multi_node_weighted_sampler.py

```diff
@@ -201,6 +201,22 @@ def get_state(self) -> Dict[str, Any]:
 
 
 class _WeightedSampler:
+    """A weighted sampler that samples from a list of weights.
+
+    The class implements the state using the following keys:
+    - g_state: The state of the random number generator.
+    - g_rank_state: The state of the random number generator for the rank.
+    - offset: The offset of the batch of indices.
+
+    Parameters:
+        weights (Dict[str, float]): A dictionary of weights for each source node.
+        seed (int): The seed for the random number generator.
+        rank (int): The rank of the current process.
+        world_size (int): The world size of the distributed environment.
+        randon_tensor_batch_size (int): The batch size for the random tensor. Default is 1000.
```

Comment:
1. spelling of random.
2. random_tensor_batch_size (int): The batch size for the random tensor generated by sampling dataset indices based on weights. Default is 1000.

Reply:
```suggestion
        randon_tensor_batch_size (int): Generating random numbers in batches is faster than individually. This setting controls the batch size, but is invisible to users and shouldn't need to be tuned. Default is 1000.
```